### DISCUSSION TITLE
The entitlements in my app bundle signature do not match.........

### DISCUSSION THREAD

**Discussion by kanisuki** [airsdk/Adobe-Runtime-Support#3462] [Q&A]
Please advise me.
I　compiled iOS app and uploaded it with Transporter.
But the Transporter showed an error.
I made p12 and mobileprovision with the same certificate in the key chain.
The message of the error,
Asset validation failed (90164)
Invalid Code signing Entitlements. The entitlements in your app bundle signature do not match the ones that are contained in the provisioning profile. According to the provisioning profile, the bundle contains a key value that is not allowed:
'＊＊＊＊＊＊＊App' for the key 'application-identifier' in 'Payload/89aa5b50306e30aa30a630e08fd4305782114f1a8371.app/
89aa5b50306e30aa30a630e08fd4305782f14f1a8a71' (ID: d52a238c-c29b-4395-a561-644006668387)

**Comment by ajwfrost** [comment]
This is most likely to be a mismatch in something like the 'get-task-allow' setting or similar .. although your message sounds like it might be the application identifier, is your Application ID in the app descriptor file definitely set to something permitted by your provisioning profile?

thanks

**Reply by kanisuki** [reply]
Other apps did not show this error, so I will check and recompile this app.

Thanks.
